infra/src/database.ts

```diff
@@ -46,6 +57,205 @@ const dbInstance = new aws.rds.Instance(nm("app"), {
   finalSnapshotIdentifier: nm("app-db-final-snapshot"),
 });
 
+const encryptedDbInstance = new aws.rds.Instance("app-encrypted", {
+  username: "openpipe",
+  password: password.result,
+  dbName,
+  instanceClass: "db.m7g.2xlarge",
+  allocatedStorage: 1210,
+  maxAllocatedStorage: 2000,
+  storageType: "io1",
+  iops: 12000,
+  engine: "postgres",
+  engineVersion: "15.5",
+  parameterGroupName: "default.postgres15",
```

Comment:
As long as we're making big db changes we should upgrade to pg 16 at the same time.

Reply:
Upgraded and addressed all other changes!